resources/keb/values.yaml

```diff
@@ -120,7 +120,6 @@ broker:
     memory: false
   events:
     enabled: false
-  enableShootAndSeedSameRegion: "false"
```

Comment:
I think this line might got removed accidentally, right?

Reply:
you are right, fixed